### PR TITLE
Added ability to filter lists and hide empty ones

### PR DESCRIPTION
**Pull request by jampez77**
This is my first commit to a public repo so no doubt I'll have done this 'wrong' in some fashion. Apologies for that!

I have added a `show_empty` boolean that stops lists without items from being displayed.
Additionally I have added `show_lists` which will only show specified list from an array of list IDs from OG. It will then also apply the logic from `show_empty`.

The one thing I could not figure out was generating the source map so I've left it. That being said it does work fine for my HA instance. I also had to change the git url referenced in order to get the correct code deploy on HA.

### CODE REVIEW

File: dist/our-groceries-card.js [hunk 1]
```diff
@@ -13298,7 +13298,7 @@ try {
...
+module.exports = JSON.parse("{\"name\":\"our-groceries-card\",\"version\":\"1.5.10\",\"description\":\"An Our Groceries card for Home Assistant Lovelace UI\",\"keywords\":[\"home-assistant\",\"homeassistant\",\"hass\",\"automation\",\"lovelace\",\"Our Groceries\",\"custom-cards\"],\"repository\":\"git@github.com:jampez77/our-groceries-card.git\",\"author\":\"Jamie Pezone <jampez77@gmail.com>\",\"license\":\"MIT\",\"dependencies\":{\"@babel/polyfill\":\"^7.4.4\",\"lit-element\":\"^2.2.0\"},\"devDependencies\":{\"@babel/cli\":\"^7.5.5\",\"@babel/core\":\"^7.5.5\",\"@babel/preset-env\":\"^7.5.5\",\"babel-loader\":\"^8.0.6\",\"eslint\":\"^6.0.1\",\"eslint-config-airbnb-base\":\"^13.2.0\",\"eslint-plugin-import\":\"^2.18.2\",\"webpack\":\"^4.36.1\",\"webpack-cli\":\"^3.3.6\",\"webpack-merge\":\"^4.2.1\"},\"scripts\":{\"lint\":\"eslint ./src\",\"start\":\"webpack --watch --config webpack/config.dev.js\",\"build\":\"webpack --config webpack/config.prod.js\"}}");
```
Comment (ljmerza) [inline dist/our-groceries-card.js]: dont include this file. i'll build when a new release is made

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "our-groceries-card",
-    "version": "1.4.0",
+    "version": "1.5.10",
```
Comment (ljmerza) [inline package.json]: dont change this file/ ill update the version.. keep repo and author the same

File: src/index.js [hunk 3]
```diff
@@ -187,24 +187,57 @@ class OurGroceriesCard extends LitElement {
...
-            <span @click=${() => this.openList(list)}>${list.name}</span>
-          </td>
-          <td class='td td-count'>
-            ${list.activeCount} 
-          </td>
-        </tr>
-        ${addingItem.show ? this.renderNewItem(addingItem, list): null}
-        ${isOpen && listDetails ? this.renderList(listDetails) : null}
-      `;
+      let listsHaveBeenSpecified = (this.config.show_lists.length > 0);
```
Comment (ljmerza) [inline src/index.js:195]: pull things like this out of the loop. they wont change each loop. make `const` if not being changed

**Review by ljmerza — CHANGES_REQUESTED** [review]

File: dist/our-groceries-card.js (hunk 1 above)
Comment (jampez77) [reply dist/our-groceries-card.js]: Reverted this info back.

File: package.json (hunk 2 above)
Comment (jampez77) [reply package.json]: Reverted this info back.

File: src/index.js (hunk 3 above)
Comment (jampez77) [reply src/index.js:195]: I've moved 'listsHaveBeenSpecified' & 'allowingEmptyLists' out of the loop.

**Comment by jampez77** [comment]
I'm just checking that i've done what I need to RE the requested changes. I think I've pressed the correct button but thought I'd check as it's been a month.

**Pull request closed without merging**